plutarch.cabal

```diff
@@ -174,7 +177,9 @@ test-suite examples
     , plutus-ledger-api
     , plutus-tx
     , rank2classes       >=1.4.4
+    , serialise
     , tasty
+    , tasty-golden
```

Comment:
We're using sydtest. Why are we adding this?

Reply:
meant to remove this